web/src/user_group_edit.js

```diff
@@ -101,6 +101,31 @@ function update_add_members_elements(group) {
     }
 }
 
+function update_general_settings_elements(group) {
+    if (!is_editing_group(group.id)) {
+        return;
+    }
+
+    // We are concerend with the General tab for changing group permissions.
+    const $group_permission_settings = $("#group_permission_settings");
+
+    // Otherwise, we adjust whether the widgets are disabled based on
+    // whether this user is authorized to change the group settings.
+    const $permission_dropdown_element = $group_permission_settings.find(".dropdown-widget-button");
+
+    if (settings_data.can_edit_user_group(group.id)) {
+        $permission_dropdown_element.prop("disabled", false);
+        $permission_dropdown_element.closest(".input-group")._tippy?.destroy();
```

Comment:
I think we can add the tippy tooltip on the button element itself. No need to apply it to `.input-group`?

Currently it results in the tooltip even if we are away from the actual dropdown element:
<details><summary>Image</summary>
<img src="https://github.com/user-attachments/assets/8cfe1e2e-0348-4bcb-9fa3-2d49f6e11aac">
</img>
</details>

Reply:
I don't think tippy instances can be assigned to a button, so I assigned them to input-group. I guess I'll try to put a container around the button.

Reply:
seems like tippy instance can't be assigned to a button. One option is maybe to use `dropdown_widget_wrapper.hbs` so that I can assign the tippy instance to the `{{widget_name}}_widget_wrapper` but then it won't have the label.

A temporary solution I guess is to just leave it until we add the `tip` box like in other organization settings, but not sure what the right solution is.

Reply:
> seems like tippy instance can't be assigned to a button.

The first example on [tippyjs homepage](https://atomiks.github.io/tippyjs/) is for a button, so that's not quite correct :)

Probably it is due to we're trying to do things in bulk i.e. on all dropdowns. See `initialize_disable_btn_hint_popover` and check what's happening with the jquery object we're passing.

Reply:
The tooltip will not appear if added on a disabled button, as they do not show hover effects due to being disabled. So, we have been following this approach of using a container div element to add the tooltip for such cases.

But yeah we need to make sure that the tooltip is not visible when hovering away from the dropdown button.